src/webui/www/private/scripts/prop-files.js

```diff
@@ -210,9 +210,9 @@ window.qBittorrent.PropFiles = (function() {
         for (let i = 0; i < options.length; ++i) {
             const option = options[i];
             if (parseInt(option.value) === priority)
-                option.setAttribute('selected', '');
+                option.selected = true;
             else
-                option.removeAttribute('selected');
+                option.selected = false;
```

Comment:
Is it really correct? Doesn't it contract the following?
>All that matters is that if the attribute is present at all, regardless of its actual value, its value is considered to be true. The absence of the attribute means its value is false.

Reply:
It doesn't appear to be doing it. Setting `selected` to `false` instead of `true` is making the bug reappear. Looking at MooTools it is simply setting booleans to `false` when "removing" them. My interpretation is that the presence of the attribute only matters when loading the DOM and setting the initial value of the property. Should the attribute be present `selected` is set to `true` regardless of the value of the attribute, but for the property that does not hold true as it is a simple boolean.

Reply:
In the [first commit](https://github.com/qbittorrent/qBittorrent/pull/19024/commits/45775327fd6d6c68b2b3e2f0301ad305a0f21511), `removeProperty()` is used. So following [mootools footsteps](https://github.com/mootools/mootools-core/blob/master/Source/Element/Element.js#L761-L763) instead of setting to `false`, shouldn't you set it to `null`?

Reply:
> Setting `selected` to `false` instead of `true` is making the bug reappear.

How about sticking to `removeAttribute()` in this case? would it work?

Reply:
i think I had a brain fart when writing the sentence you quoted, but the change as suggested by my pull request appears to be working properly. I tried to again to be sure. My interpretation of the behavior of the browser also remains unchanged.

Reply:
@glassez 
Despite the quoted text from [your post](https://github.com/qbittorrent/qBittorrent/pull/19024#discussion_r1208552441) I think we can land this PR in master and give it a try.

Reply:
> @glassez Despite the quoted text from [your post](https://github.com/qbittorrent/qBittorrent/pull/19024#discussion_r1208552441) I think we can land this PR in master and give it a try.

I just wanted to draw your attention to something that seemed suspicious to me. I'm not blocking this PR. As I said earlier, WebUI is beyond my competence. So do with it as you see fit.